sycl/source/detail/config.hpp

```diff
@@ -127,28 +127,6 @@ template <ConfigID Config> class SYCLConfig {
   }
 };
 
-template <> class SYCLConfig<SYCL_PI_TRACE> {
```

Comment:
`SYCL_PI_TRACE` should print a deprecation message now, mentioned in `sycl/source/detail/global_handler.cpp` file.

Reply:
> `SYCL_PI_TRACE` should print a deprecation message now, mentioned in `sycl/source/detail/global_handler.cpp` file.

I think that you forgot to push the change, because I don't see the file you mentioned being updated in this PR

Reply:
It should be added with one of the PR commits, the updated file with the changed part is [here](https://github.com/intel/llvm/blob/a627db5c517923b893cfa943d49e6dbad38b0886/sycl/source/detail/global_handler.cpp#L360).